src/main/scala/Resolver.scala

```diff
@@ -23,7 +23,7 @@ case class BintrayMavenRepository(
       Duration.Inf) match {
         case (201, _) =>
         case (_, fail) =>
-          println(fail)
```

Comment:
Yes, please, do so :) In fact, you should do `log.error(fail)`. With the current statement, you will prevent the `throw new RuntimeException` from being executed, and that's the one that tells the user what really happened.